styles.css

```diff
@@ -1 +1,120 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+.container {
+    display: grid;
+    color: white;
+    padding: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px 50px;
+    grid-template-areas:
+        "a"
+        "b"
+        "c"
+        "d"
+        "e"
+        "."
+        "f";
+    row-gap: 16px;
+    margin: 10px;
+
+}
+
+.item {
+    background-color: rgb(186, 59, 70);
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+
+#blok-a {
+    grid-area: a;
+}
+
+
+#blok-b {
+    grid-area: b;
+    padding: 20px;
+}
+
+
+#blok-c {
+    grid-area: c;
+}
+
+
+#blok-d {
+    grid-area: d;
+}
+
+
+#blok-e {
+    grid-area: e;
+}
+
+
+#blok-f {
+    grid-area: f;
+}
+
+
+
+@media (min-width: 992px) {
+
+    .container {
+        display: grid;
```

Comment:
.container in de media-query is goed, staan wat dingen in die hij al inherit van de mobile versie die niet echt dubbel hoeven, maar ja, verder gewoon prima dus